tox.ini

```diff
@@ -150,6 +150,8 @@ setenv =
 extras = test
 deps =
     {[plotting]deps}
+; recreating the environment to avoid dependency conflict when not starting from a clean slate
+recreate = true
```

Comment:
Is there any cost to that ?

Reply:
Not really. 

It means that using tox to run those tests will have some overhead at start up every time: but it is only run once a week in CI and the extra over-head is small compare to the run time of the entire test suite.

It could become be annoying if we start to need to debug things manually more often. 
I had to do that here, then in this case setting up one's own environment as suggested by @ymzayek makes sense.

But given that the numpy2 compatibility workflow won't be with us forever, I would suggest not to worry and only revisit if this kind friction becomes annoying for developers.